### PR TITLE
Create issue_backlog.md

### PR DESCRIPTION
**Pull request by pwittrock**
Doc for describing how issues should be picked for new contributors.

### CODE REVIEW

**Comment by fabianofranz** [comment]
@pwittrock Do you mind checking the "Allow edits from maintainers" here on the right if it's available? So that I can adjust minor things directly and don't bother you with minor nits.  ;)

**Comment by pwittrock** [comment]
@fabianofranz I can't find the button...

File: docs/maintainers/issue_backlog.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+- Reducing the need for constant communication by having the work be well defined and clearly scoped in the issues
+  themselves.
+- Ensure each issue has a stake holder that is committed to seeing it through.  This individual will be motivated to
+  make sure it is reviewed.
+
+## Contribution lifecycle
+
+1. Issue is created with description and labels
+  - Made sure the issue meets guidelines for a good issue
+  - Issue must have a stakeholder invested in the work being completed
```
Comment (monopole) [inline docs/maintainers/issue_backlog.md]: Any ideas on how the stakeholder / person-who-is-willing-to-review-PR identified?

File: docs/maintainers/issue_backlog.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+2. SIG agrees that work for issue will be accept
+  - Must be low overhead approach
```
Comment (monopole) [inline docs/maintainers/issue_backlog.md]: The list elements are messed up here. Try three space indent:

1. foo
   * alice
   * bob
1. canada
1. alligator

File: docs/maintainers/issue_backlog.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+- Documenting functions with their purpose and usage
+
+### Test coverage
+
+Improving test coverage and augmenting e2e tests with integration tests is also a good candidate for 1st and
+2nd time contributors.  Writing tests for libraries requires understanding how they behave and are meant to be used. 
+Improving code coverage allows the project to move more quickly by reducing regressions issues that the SIG must field,
+and by providing a safety net for code reviewers ensuring changes don’t break existing functionality.
+
+- Write unit tests for functionality only covered by integration and e2e tests
```
Comment (monopole) [inline docs/maintainers/issue_backlog.md]: please provide link to kubernete's distinction between integration and e2e test.

File: docs/maintainers/issue_backlog.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+- Write integration tests for functionality only covered by e2e tests
+- Improve coverage for edge cases and different inputs to functions
+- Checking behavior for invalid arguments
+- Refactoring existing tests to pull out common code into reusable functions
+  - **Note:** this should be very well scoped as it impacts existing tests and reviewers need to make sure nothing breaks
+
+### Encapsulated libraries
```
Comment (monopole) [inline docs/maintainers/issue_backlog.md]: Isn't this just "Libraries"?

**Comment by pwittrock** [comment]
@fabianofranz I think the problem is I created the PR through the UI

**Comment by pwittrock** [comment]
PTAL
